test/reporters/tests/html.test.ts

```diff
@@ -47,7 +47,9 @@ describe('html reporter', async () => {
     const metaJsonGzipped = fs.readFileSync(resolve(root, `${basePath}/html.meta.json.gz`))
     const metaJson = zlib.gunzipSync(metaJsonGzipped).toString('utf-8')
     const indexHtml = fs.readFileSync(resolve(root, `${basePath}/index.html`), { encoding: 'utf-8' })
-    const resultJson = parse(metaJson.replace(new RegExp(vitestRoot, 'g'), '<rootDir>'))
+    const resultJson = parse(metaJson.replace(new RegExp(vitestRoot, 'g'), '<rootDir>'), (_key, value) => {
+      return typeof value === 'string' ? stripAnsi(value) : value
+    })
```

Comment:
`NO_COLOR` was also affecting html reporter, so I stripped it just for this test case.